resource_calendar_rrule/models/resource_calendar.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import datetime
+import pytz
+from dateutil.relativedelta import relativedelta
+from dateutil.rrule import WEEKLY
+from odoo import _, api, exceptions, fields, models, tools
+
+DEFAULT_MORNING_HOUR = 8.0
+DEFAULT_AFTERNOON_HOUR = 13.0
+
+
+class ResourceCalendar(models.Model):
+    _inherit = 'resource.calendar'
+
+    simplified_attendance = fields.Serialized(
+        'Working time', compute='_compute_simplified_attendance',
+        inverse='_inverse_simplified_attendance',
+    )
+
+    @api.model
+    def default_simplified_attendance(self):
+        result = {
+            'type': 'all',
+            'data': [],
+            'start': fields.Date.today(),
+            'stop': False,
+        }
+        for day in [0, 1, 2, 3, 4]:
+            result['data'].extend([
+                {
+                    'day': day,
+                    'morning': 4,
+                    'afternoon': 4,
+                },
+            ])
+        return result
+
+    @api.multi
+    def _compute_simplified_attendance(self):
+        for this in self:
+            for attendance in this.attendance_ids:
+                this.simplified_attendance = this\
+                    ._simplified_from_attendances()
+
+    @api.model
+    @api.depends('attendance_ids')
+    def _simplified_from_attendances(self):
+        self.ensure_one()
+        result = {'type': 'all', 'data': [], 'data_odd': []}
+        for day in [0, 1, 2, 3, 4, 5, 6]:
+            result['data'].append({
+                'day': day,
+                'morning': 0,
+                'afternoon': 0,
+            })
+            result['data_odd'].append({
+                'day': day,
+                'morning': 0,
+                'afternoon': 0,
+            })
+        warn = ''
+        start = datetime.datetime.today().replace(tzinfo=pytz.utc)
+        stop = False
+        for attendance in self.attendance_ids:
+            for rule in attendance.rrule._rrule:
+                if rule._byeaster or rule._bymonth or rule._bymonthday or\
+                        rule._byyearday or rule._interval not in [1, 2] or\
+                        rule._freq != WEEKLY or not rule._dtstart:
+                    # TODO: this neds to warn somehow if the rrule can't be
+                    # simplified
+                    warn += _('Ignoring %s') % rule
+                    continue
+
+                if rule._dtstart < start:
+                    start = rule._dtstart
+                if rule._until and (not stop or stop > rule._until):
+                    stop = rule._until
+
+                odd = all(
+                    bool(rule._dtstart.isocalendar()[1] % 2)
+                    for occurrence in rule[:2]
+                )
```

Comment:
@hbrunn Turns out this goes wrong in cases like December 2009 - January 2010 where there is 'week 53' followed by 'week 1'. There needs to be an extra check on `_interval` being 2.